### PR TITLE
Adding example from discord

### PR DESCRIPTION
**Pull request by ghilac**

### CODE REVIEW

File: CVLByExample/ReadOnlyReentrancy/certora/conf/BuiltinROReentrancyVulnerableBankOther.conf [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (sitvanit83) [inline CVLByExample/ReadOnlyReentrancy/certora/conf/BuiltinROReentrancyVulnerableBankOther.conf:1]: What does "other" mean in the name. Can you give a more explanatory name?

File: CVLByExample/ReadOnlyReentrancy/contracts/VulnerableBankOther.sol [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    }
+
+    function getEthBalance() external view returns(uint){
+        return address(this).balance;
+    }  
+    function withdraw(uint _amount) public {
+        if (balances[msg.sender] >= _amount) {
+            (bool result,) = msg.sender.call{value:_amount}("");
+            if (result) {
+                _amount;
```
Comment (sitvanit83) [inline CVLByExample/ReadOnlyReentrancy/contracts/VulnerableBankOther.sol:23]: What does this line (_amount) do?

File: CVLByExample/ReadOnlyReentrancy/contracts/VulnerableBankOther.sol [hunk 3]
```diff
@@ -0,0 +1,30 @@
+import "@openzeppelin/contracts/utils/math/SafeMath.sol";
+
+contract VulnerableBankOther {
+  
+    using SafeMath for uint256;
+    mapping(address => uint) public balances;
```
Comment (sitvanit83) [inline CVLByExample/ReadOnlyReentrancy/contracts/VulnerableBankOther.sol:6]: uint should be replaced with uint256 (also in the other contracts).

File: CVLByExample/ReadOnlyReentrancy/contracts/VulnerableBankOther.sol [hunk 4]
```diff
@@ -0,0 +1,30 @@
+import "@openzeppelin/contracts/utils/math/SafeMath.sol";
+
+contract VulnerableBankOther {
+  
+    using SafeMath for uint256;
```
Comment (sitvanit83) [inline CVLByExample/ReadOnlyReentrancy/contracts/VulnerableBankOther.sol:5]: This is the main difference from BulnerableBank.sol?

File: CVLByExample/ReadOnlyReentrancy/certora/conf/BuiltinROReentrancyVulnerableBankOther.conf (hunk 1 above)
Comment (ghilac) [reply CVLByExample/ReadOnlyReentrancy/certora/conf/BuiltinROReentrancyVulnerableBankOther.conf:1]: yeah I have an idea

File: CVLByExample/ReadOnlyReentrancy/contracts/VulnerableBankOther.sol (hunk 2 above)
Comment (ghilac) [reply CVLByExample/ReadOnlyReentrancy/contracts/VulnerableBankOther.sol:23]: actually this is the original code, I did not touch it. It's pretty uncommon https://stackoverflow.com/questions/70767898/what-does-mean-expression-variablename-in-solidity
I even found a stack overflow issue about it... 
I can fix it or not, wdyt?

File: CVLByExample/ReadOnlyReentrancy/contracts/VulnerableBankOther.sol (hunk 3 above)
Comment (ghilac) [reply CVLByExample/ReadOnlyReentrancy/contracts/VulnerableBankOther.sol:6]: as I said, this is their original code. Should I fix that?

File: CVLByExample/ReadOnlyReentrancy/contracts/VulnerableBankOther.sol (hunk 4 above)
Comment (ghilac) [reply CVLByExample/ReadOnlyReentrancy/contracts/VulnerableBankOther.sol:5]: yeah not really a lot of difference

**Comment by nd-certora** [comment]
duplicate

**Pull request closed without merging**